packages/terra-overlay/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Focus changes to container overlay text for screen reader compatibility
+* Main page content hidden from screen reader while fullscreen overlay is deployed
 
```

Comment:
Lets add : 
```
### Added
- Added react-portal dependency for ...
- Added `rootSelector` prop for ...
```

Reply:
Added during the slash and burn master rebase! :)